.github/workflows/ci-test.yml

```
@@ -0,0 +1,35 @@
+    if: success()
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: 자바 21 설치
+        uses: actions/setup-java@v4
+        with:
+          java-version: '21'
+          distribution: 'temurin'
+
+      - name: Gradle 캐싱
+        uses: actions/cache@v3
```

actionlint (1.7.10)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)